plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/BaseKuduWithEmptyInferSchemaConnectorSmokeTest.java

```diff
@@ -19,7 +19,7 @@
 
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 
-public abstract class AbstractKuduWithEmptyInferSchemaConnectorTest
+public abstract class BaseKuduWithEmptyInferSchemaConnectorSmokeTest
         extends AbstractKuduConnectorTest
```

Comment:
The class combination `BaseKuduWithEmptyInferSchemaConnectorSmokeTest` & `AbstractKuduConnectorTest` is incorrect. I know it's changed to `BaseKuduConnectorSmokeTest` in the 3rd commit, but the 2nd commit fails on `BaseConnectorTest.ensureTestNamingConvention`.

Reply:
@ebyhr Should I stash the 2nd and 3rd commits, then? Like I did before. But the downside of doing so is Git will think I deleted `AbstractKuduWithEmptyInferSchemaConnectorTest` and created `BaseKuduWithEmptyInferSchemaConnectorSmokeTest`, as described in my first comment..

Reply:
@ebyhr Thanks! Didn't notice that before. Just stashed the 2nd and 3rd commits (like I did before) so that none of the commits fails `BaseConnectorTest.ensureTestNamingConvention`. The downside of doing so is Git will think I deleted `AbstractKuduWithEmptyInferSchemaConnectorTest` and created `BaseKuduWithEmptyInferSchemaConnectorSmokeTest`, though, as described in my first comment..